common/internal-types/Cargo.toml

```diff
@@ -13,7 +13,7 @@ crate-type = ["rlib"]
 
 [dependencies]
 aquamarine = { workspace = true }
-bloomfilter = { workspace = true }
+bloomfilter = { workspace = true, features = ["serde"] }
```

Comment:
This should be under the feature `serde` of this package.

Reply:
However, Bloom is also used in "TagBloomFilter" implementation in default

Reply:
That's okay, but serde is only used when performing write and read to the disk.